server/routes/favoritesRoutes.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+
+import { getPlantsFromFavorite, addPlantToFavorite, removePlantFromFavorite } from '../controllers/favoritesController';
+
+const router = express.Router();
+
+router.get('/', getPlantsFromFavorite);
+router.post('/', addPlantToFavorite);
+router.delete('/:plantId', removePlantFromFavorite);
```

Comment:
For the consistency changes you would want to remove the variable param here too:

```suggestion
router.delete('/', removePlantFromFavorite);
```